lib/services/db.test.js

```diff
@@ -156,6 +156,23 @@ describe(_.startCase(filename), function () {
         expect(str).to.equal('{}');
       });
     });
+
+    it('can stream an object instead of a JSON string', function (done) {
+      const onData = (data) => expect(data).to.deep.equal({key: '1', value: '2'}),
+        onEnd = () => done(),
+        onError = done;
+
+      return lib.put('1', '2')
+        .then(()=>{
+          const pipe = fn({
+            json: false
+          });
+
+          pipe.on('data', onData);
+          pipe.on('end', onEnd);
+          pipe.on('error', onError);
+        });
+    });
   });
```

Comment:
Note: This new test is needed because without sitemap tests istanbul will  complain about `db.list` never running with `json` set to `false`.